### PR TITLE
Added Test Cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
Ref Tickets:
https://mattermost.atlassian.net/browse/MM-44348
https://mattermost.atlassian.net/browse/MM-46337
https://mattermost.atlassian.net/browse/MM-23837
https://mattermost.atlassian.net/browse/MM-46529
- [x] Browser
- [x] Desktop App (version, OS)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/60"></a>

### CODE REVIEW

**Review by yasserfaraazkhan — APPROVED** [review]
🎉 


just a small request to change `channelA` -> `Channel A`, `userA` -> `User A` like wise. (in most test case this is spelt that way.)

**Review by saturninoabril — APPROVED** [review]
Thanks @furqanmlk, LGTM.

File: data/test-cases/integrations/MM-Test-case-1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+# (Required) Ensure all values are filled up
+name: " Multiple users/channels in invite slash command"
```
Comment (saturninoabril) [inline data/test-cases/integrations/MM-Test-case-1.md:3]: Test case saved as MM-T5388

File: data/test-cases/suite/activity-and-insights/insights/my-insights/MM-test-case-1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+# (Required) Ensure all values are filled up
+name: "Insights - My insights - emoji tooltip to Top Reactions"
```
Comment (saturninoabril) [inline data/test-cases/suite/activity-and-insights/insights/my-insights/MM-test-case-1.md:3]: Test case saved as MM-T5389

File: data/test-cases/suite/custom-groups/MM-test-case-1.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+# (Required) Ensure all values are filled up
+name: "Show number of group members in group autocomplete"
```
Comment (saturninoabril) [inline data/test-cases/suite/custom-groups/MM-test-case-1.md:3]: Test case saved as MM-T5390

**Pull request merged**